util/test/demos/CMakeLists.txt

```diff
@@ -73,8 +87,14 @@ if(BUILD_ANDROID)
     # Default to arm64 if nothing is specified on the command line.
     # Options are {armeabi-v7a,arm64-v8a}
     set(ANDROID_ABI "arm64-v8a" CACHE STRING "The Android ABI to build for")
-    
+
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DANDROID=1")
+
+    # The version of fmt used will not build due to std::char_traits support of non-std types being
+    # deprecated in the version of LLVM used by NDK v26
+    if("${ANDROID_NDK_VERSION}" VERSION_GREATER_EQUAL "26.0.0")
```

Comment:
I got bitten by these version compares recently, cmake's `VERSION_GREATER_EQUAL` was added in a later version than what we support. You need to formulate this as `NOT X VERSION_LESS Y` instead :(.

Reply:
Ah yes, you're right it was brought in for v3.7.